ocamlformat-rpc-help.txt

```diff
@@ -41,6 +45,16 @@ COMMANDS
        by OCamlFormat, the formatted output is sent as a reply of the same
        form Format CSEXP
 
+       Specific commands supported on version v2 are:
+
+       - Format CSEXP: submits a list as canonical s-expression CSEXP, where
+       the first element of the list is a string to be formatted by
+       OCamlFormat. The other arguments are (key, value) pairs, where key can
+       be either "Path", "Config", "OverridingPath" or "OverridingConfig".
```

Comment:
There's no `Path` and `Config` key anymore. I think the `Overriding` prefix can be removed everywhere.

Reply:
done